src/values/blendMode.ts

```diff
@@ -37,16 +37,40 @@ function overlay():BlendModeQualifier {
   return new BlendModeQualifier('overlay');
 }
 
+/**
+ * @memberOf Values.BlendMode
+ * @description Add an overlay image blended using the 'mask' blend mode.
```

Comment:
Ads instead of Add?

Reply:
I'll leave it for the docs team,
In practice this function doesn't really add anything, it just sets the blendMode of the overlay.